src/BrowserInterop/BrowserInterop.csproj

```diff
@@ -6,7 +6,7 @@
     <PackageId>BrowserInterop</PackageId>
     <Authors>Rémi BOURGAREL</Authors>
     <Company>Rémi BOURGAREL</Company>
-    <PackageVersion>1.1.1</PackageVersion>
+    <PackageVersion>1.1.1.13</PackageVersion>
```

Comment:
You can remove this I'll do a patch release I think (1.1.2) with an other bug fix, then a minor upgrade for .net 5 (1.2)

Reply:
I just need that versioning for local pack and wildcard based restore, to get automatically the latest version I've built locally. I used the 4th digit to automatically 1.1.2 once it is released. feel free to pull the version up to 1.1.2 when you do a release.